### DISCUSSION TITLE
[Unhandled promise rejection: Error: You attempted to use a firebase module that's not install]d on your Android project by calling firebase.app().

### DISCUSSION THREAD

**Discussion by vinicius-lacerda** [invertase/react-native-firebase#6346] [Q&A]
Guys, I've been trying to solve this problem for 1 week and I can't find any solutions.

I'm trying to login with an authenticated user. It was just to show his data in console.log, to confirm that it worked. But when I enter any email and password, it returns this error.

https://github.com/vinicius-lacerda/gopizza

![firebase](https://user-images.githubusercontent.com/15632297/175992481-93c0e534-144a-4f7c-b2bb-11f281114d76.png)

**Comment by mikehardy** [comment]
It appears you are logging an issue, but conveniently for you as a discussion, which is avoiding our new issue template.

That template prompts you to include the information we need to help, and without it, we can't help

https://stackoverflow.com/help/how-to-ask

Here's a start to finish example of correctly integrating react-native-firebase, with a call to a auth() in it, you can expand on that perhaps to do other auth calls https://github.com/mikehardy/rnfbdemo/blob/main/make-demo.sh

I haven't had time to update this lately but it is actually in use by me in commercial projects. You might like it https://github.com/invertase/react-native-firebase-authentication-example

**Reply by vinicius-lacerda** [reply]
I managed to solve the problem.

I created another project from scratch with another name, install it as visible libs and every time I use 'expo start', I used yarn start and yarn android.
